vcbuild.bat

```diff
@@ -325,10 +327,12 @@ if "%target%"=="Build" (
 )
 if "%target%"=="node" if exist "%config%\cctest.exe" del "%config%\cctest.exe"
 if defined msbuild_args set "extra_msbuild_args=%extra_msbuild_args% %msbuild_args%"
+if defined ccache_path set "extra_msbuild_args=%extra_msbuild_args% /p:TrackFileAccess=false /p:CLToolPath=%ccache_path% /p:ForceImportAfterCppProps=%CD%\props_4_ccache.props"
 @rem Setup env variables to use multiprocessor build
-set UseMultiToolTask=True
-set EnforceProcessCountAcrossBuilds=True
-set MultiProcMaxCount=%NUMBER_OF_PROCESSORS%
+set "extra_msbuild_args=%extra_msbuild_args% /p:UseMultiToolTask=True"
+set "extra_msbuild_args=%extra_msbuild_args% /p:EnforceProcessCountAcrossBuilds=True"
+set "extra_msbuild_args=%extra_msbuild_args% /p:MultiProcMaxCount=%NUMBER_OF_PROCESSORS%"
+set "extra_msbuild_args=%extra_msbuild_args% /p:VcpkgEnabled=false"
```

Comment:
This line was not here before (the other 3 are rewritten). Is this connected with your [other PR](https://github.com/nodejs/node/pull/52181/files)?

Reply:
Partially, if this pull request is approved, the <https://github.com/nodejs/node/pull/52181/files> will be upgraded to use vcbuild  ccache  switch directly:)

I'm a little worried, this pr changed too much, which is hard to accept, so I decided to split it out or drop this pull request, but use cmdline to config msbbuild directly, which is enough to make it work.

Btw, I don't like vcbuild.bat, why not use [configure.py](https://github.com/nodejs/node/blob/main/configure.py) directly, The windows support for developing node itself is not so good, maybe most guys use linux/mac:)

Reply:
BTW, maybe that's my fault, it seems just setting environment variable is enough, I will check it later

https://learn.microsoft.com/en-us/visualstudio/msbuild/how-to-use-environment-variables-in-a-build?view=vs-2022

Reply:
@StefanStojanovic Hi Stefan, sorry about my fault, it works by setting msbuild property directly

I do a test

```bat
set TrackFileAccess=false
set UseMultiToolTask=true
: allow link symbols redefine
set ForceImportAfterCppTargets=some_build.targets
msbuild ConsoleApplication_with_symbol_redefine.sln
```
and it works!